repository/managers.py

```diff
@@ -110,7 +110,46 @@ def collection_items(self, groupId, collectionId, page=1):
         }
 
 
-    def item(self, groupId, itemId):
+    def item_files(self, groupId, itemId):
+        """
+        Fetch individual item from repository's endpoint and list all associated files for import.
+
+        Args:
+            groupId: The group ID in the repository
+            itemId: The item ID in the repository
+
+        Returns:
+            A dictionary containing a list of files with their respective details.
+        """
+        headers = auth.citesphere_auth(self.user, self.repository)
+        url = f"{self.repository.endpoint}/api/v1/groups/{groupId}/items/{itemId}/"
+        response = requests.get(url, headers=headers)
+
+        if response.status_code == 200:
+            item_data = response.json()
+
+            files = []
+            
+            # Extract Giles upload file details if available
+            giles_uploads = item_data.get('item', {}).get('gilesUploads', [])
+
+            if giles_uploads:
```

Comment:
if a file is still being process, there will only be a progress id, not an extracted text. Vogon should probalby indicate a file is still being processed.